examples/nextjs-blog/pages/blog/post.js

```diff
@@ -17,29 +14,26 @@ class Post extends PureComponent {
     }
   }
 
-  componentDidMount() {
-    // todo: this isn't so lekker because the path may change in future. is it
-    // possible to share the `query.slug` value from the server's execution with
-    // the client?
+  async componentDidMount() {
     const { pathname } = location
-    const slug = pathname.replace('/blog/', '')
+    const slug = pathname.split('/').pop()
 
+    // todo: SDK bug - subscribing using an event type doesn't work
```

Comment:
Hey @jordanch what is currently happening?

Reply:
@jperasmus Bug logged against SDK Epic. Inline here for convenience: 

JS SDK v1 > Content > subscribe > subscribe with changeType doesn't call callback on change

https://responsivestudio.atlassian.net/browse/FL-1514

When adding the changeType option, the callback doesn't fire on document event.

Reply:
Thanks for logging, I'll check it out.